Removed stale unused copy of u_kept_successful_command body

nickanderson opened the pull request:
This change removes body classes u_kept_successful_command from the standandard
library. Note an identically named classes body still exists inside the
update.cf policy entry. This body that is being removed was only loaded by the
promises.cf entry and is believed to be some left over artifact from a time when
promises.cf loaded the update.cf policy. Another classes body,
kept_successful_command still remains in the standard library, and should be
used instead.

Ticket: CFE-3617
Changelog: Removed stale unused copy of u_kept_successful_command body. If you receive an error about undefined body, alter your policy to use kept_successful_command instead.

craigcomstock reviewed (commented):
typo in commit message

```
from the standandard
```

s/standandard/standard/

:)

nickanderson commented:
Nice, thanks! fixed.

> typo in commit message
> 
> ```
> from the standandard
> ```
> 
> s/standandard/standard/
> 
> :)

Nice, thanks, fixed.
